src/components/FormControl/FormControl.tsx

```diff
@@ -0,0 +1,22 @@
+import InputLabel from "@material-ui/core/InputLabel";
+import MenuItem from "@material-ui/core/MenuItem";
+import FormControl from "@material-ui/core/FormControl";
+import Select from "@material-ui/core/Select";
+import React from "react";
+
+const FormControlField = (props: any) => {
```

Comment:
nie wrzucaj wszystkich propsów jako jeden tylko używaj destrukturyzacji obiektów czyli
```
const FormControlField = ({
 inputLabel,
 value,
 onChange,
}) => {
```
Kolejna rzecz o której wspominałem Monice w review to żeby pomijać słowo return jak w arrow function nie masz nic poza zwracaniem bo wtedy return jest domyślny czyli będzie coś takiego w skrócie:
```
const FormControlField = ({
 inputLabel,
 ...
}) => (
<FormControl>
  ...
</FormControl>
);
```
a tak poza tym to ten plik jest średnio potrzebny bo to jest użyte tylko raz i choćby przez `style={{ width: "30%" }}` nie wygląda na super reużywalne - ja bym to dał bezpośrednio tam gdzie jest użyte i usuną ten plik.

Reply:
aa sorry teraz zauważyłem, że tej funkcji używasz tam kilka razy, to wtedy przenieś ten plik obok tego gdzie jest użyty jeśli jest tylko tam używany

Reply:
Jak przeniesc obok? FormControlField jest wyraznie stateless componentem, nie powinnien sie znajdowac w folderze `containers`. Jest dobrze.

Ja bym sie tu tylko przyczepil do uzywania inline style'ow, kiedy jest funkcja do tworzenia hooka do stylow makeStyles(). Widzialem to w wielu miejscach u innych ludzi, i widze to jako brak systematycznosci w projekcie. Jedni uzywaja material-ui do stworzenia odpowiednich klas ze stylami, drudzy wrzucaja inline style.

Reply:
Ale jest jest też wyraźnie komponentem przystosowanym do użycia w specyficznym miejscu, a nie w pełni reużywalnym. To generalnie zależy od organizacji w projekcie ale na pewno potrzebujemy rozdzielenia na rzeczy używane lokalnie i sherowane w całym projekcie czego teraz w takim razie brakuje. Ja zazwyczaj robie podobnie jak mamy teraz z tą różnicą, że zamiast `containers` nazywam ten folder `views` - wtedy widać odrazu że są raczej unikalne (jak wszystkie w naszym wypadku) . Skoro więc mamy unikalne widoki to jak najbardziej rzeczy które będą użyte tylko w nich powinny być obok a nie gdzieś dalej mieszać się z innymi.